documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -165,3 +165,31 @@ spec:
           key: cruise-control-log4j.properties
     # ...
 ----
+
+[[api-security-configuration]]
+[discrete]
+== Cruise Control REST API security
+
+The Cruise Control REST API is secured with HTTP Basic authentication and SSL to protect the cluster against potentially destructive Cruise Control operations, such as decommissioning Kafka brokers.
+
+We recommend that Cruise Control in Strimzi is **only used with these settings enabled**.
+You should not disable the built-in HTTP Basic authentication or SSL settings described below.
+
+* To disable the built-in HTTP Basic authentication, set `webserver.security.enable` to `false`.
+* To disable the built-in SSL, set `webserver.ssl.enable` to `false`.
+
+.Example Cruise Control configuration to disable API authorization and authentication
```

Comment:
"authorization and authentication" now it's more authentication/authorization and SSL